.gitignore

```diff
@@ -2,3 +2,4 @@
 .idea/**
 .coverage*
 coverage*
+**/__pycache__/**
```

Comment:
Hello, Indira! Thanks for adding this pycache to the .gitignore. We actually need this as the cache often conflicts between one another.